website/routes.py

```diff
@@ -337,6 +349,36 @@ def make_url_map(app):
                     notemplate
                 )
             ])
+            EXTERNAL_EMBER_APPS['ember_osf_web']['asset_prefixes'] = [
```

Comment:
Is it necessary to modify `EXTERNAL_EMBER_APPS` here? In general, we try to avoid mutating settings.